scripts/apidocgen/postprocess/main.go

```diff
@@ -132,6 +133,11 @@ func writeDocs(sections [][]byte) error {
 		})
 	}
 
+	// Sort API pages
+	sort.Slice(mdFiles, func(i, j int) bool {
+		return sort.StringsAreSorted([]string{mdFiles[i].title, mdFiles[j].title})
+	})
```

Comment:
Can we still have "General" as the first item and sort after that?

Reply:
Fixed!